src/cloud/release-notes/mcd-release-notes.md

```diff
@@ -0,0 +1,103 @@
+---
+group: cloud-guide
+title: Magento Cloud Docker release notes
+functional_areas:
+  - Cloud
+  - Setup
+  - Configuration
+---
+
+The [Magento Cloud Docker](https://github.com/magento/magento-cloud-docker) package provides functionality and Docker images to deploy {{ site.data.var.ee }} to a local Cloud environment. These release notes describe the latest improvements to this package, which is a component of [{{ site.data.var.csuite }}]({{ page.baseurl }}/cloud/release-notes/cloud-tools.html).
+
+The {{site.data.var.mcd}} package uses the following version sequence: `<major>.<minor>.<patch>`.
+
+The release notes include:
+
+-  {:.new}New features
+-  {:.fix}Fixes and improvements
+
+## v1.0.0
+
+-  {:.new}**Added a stand-alone Magento Cloud Docker package**–Moved the Docker package to the [new `magento-cloud-docker` repository](https://github.com/magento/magento-cloud-docker) to maintain code quality and provide independent releases.<!--MAGECLOUD-3986-->
+
+-  {:.new}**Added versioning to the Docker images**–You must now update the {{site.data.var.mcd}} package to get the updated images.
```

Comment:
If referring to the package, like ece-tools, usually it is formatted as inline code? If not, skip.

```suggestion
-  {:.new}**Added versioning to the Docker images**–You must now update the `{{site.data.var.mcd}}` package to get the updated images.
```

Reply:
We aren't consistent, but using inline code is better. Will sweep for that.